### ISSUE TITLE
decide on topic for last day

### ISSUE THREAD

**Issue by jaberg** [clementfarabet/ipam-tutorials#3]
I put an idea about talking about Model Selection / Hyper-parameter optimization on the last day, but basically it was for lack of anything else.

Other ideas?

**Comment by jaberg** [comment]
Could look at temporal learning algorithms - what do you have for timeseries prediction in Torch?

**Comment by clementfarabet** [comment]
Yep we could do something temporal. We have the basic ingredients to build recursive neural nets and the likes, but I don't have much experience with it. It could be fun though, more fun than hyper-parameter selection I guess. Do you know a dataset that could be interesting? Or we could just grab a bunch of stock data, and try to do some prediction.

**Comment by jaberg** [comment]
Yeah hyperparameters are interesting to me, but no fun to explore in a
hands-on session because there are a lot of tools involved and the
interesting experiments take too long.

On the Theano side, a temporal model example would have to be combined
with a little under-the-hood intro to Theano's API, which I was
planning on putting off until the last day anyway. So that's actually
shaping up in my mind: Theano + temporal modeling with theano.scan

On Sun, Jul 1, 2012 at 3:40 AM, Clement Farabet
reply@reply.github.com
wrote:

**Comment by clementfarabet** [comment]
> Yeah hyperparameters are interesting to me, but no fun to explore in a
> hands-on session because there are a lot of tools involved and the
> interesting experiments take too long.

Exactly. 

> On the Theano side, a temporal model example would have to be combined
> with a little under-the-hood intro to Theano's API, which I was
> planning on putting off until the last day anyway. So that's actually
> shaping up in my mind: Theano + temporal modeling with theano.scan

OK, same on my side, it involves a little bit of hacking, but nothing too crazy.

Let me try to work something out on that side.
